src/screens/DiscoveryScreenV2/index.js

```diff
@@ -43,9 +45,14 @@ const DiscoveryScreenV2 = ({route}) => {
   const [searchText, setSearchText] = React.useState('');
   const [isFocus, setIsFocus] = React.useState(true);
   const [isFirstTimeOpen, setIsFirstTimeOpen] = React.useState(true);
+  const [tabs, setTabs] = React.useState({Users: 0, Communities: 0, Domains: 0, News: 0});
 
-  const [, discoveryDispatch] = React.useContext(Context).discovery;
+  const [discoveryData, discoveryDispatch] = React.useContext(Context).discovery;
+  const [profileState] = React.useContext(Context).profile;
   const cancelTokenRef = React.useRef(axios.CancelToken.source());
+  const [, followingDispatch] = React.useContext(Context).following;
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `tabs` state and the `followingDispatch` context are significant changes. However, without seeing how they're used in the rest of the code, it's hard to comment on their correctness or efficiency.



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
its okay, it will always maintained in every single data changes